build.sbt

```diff
@@ -68,14 +76,18 @@ lazy val anorm = project
       // New functions
       missMeth("anorm.Sql.unsafeStatement$default$2"),
       missMeth("anorm.Sql.unsafeResultSet"),
-      missMeth("anorm.Sql.unsafeStatement")),
+      missMeth("anorm.Sql.unsafeStatement"),
+      missMeth("anorm.ToStatement.contramap"),
```

Comment:
Yes, I think it's ok to break binary compatibility in 2.6.0.

`provided` doesn't solve the problem, since the old anorm version would still need to be used for compilation.